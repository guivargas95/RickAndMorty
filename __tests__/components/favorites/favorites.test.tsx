import React from 'react';
import {render} from '@testing-library/react'
import Favorites from "../../../src/components/favorites";

describe('Favorites', () => {
  test('renders correctly', () => {
    const { container } = render(<Favorites />);

    expect(container).toMatchSnapshot();
  });
});