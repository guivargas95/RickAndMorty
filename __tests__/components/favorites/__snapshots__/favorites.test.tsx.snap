// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites renders correctly 1`] = `
<div>
  <div>
    <button>
      <img
        class="w-12 mr-auto ml-auto"
        src="/img/blackstar.png"
      />
    </button>
  </div>
</div>
`;
